workers/ohsome_quality_analyst/cli.py

```diff
@@ -210,10 +228,11 @@ def create_report(
         bpolys = None
         report = asyncio.run(
             oqt.create_report(
-                report_name=report_name,
+                report_name,
```

Comment:
My thinking was to make it clear that this is a madatory parameter. All other parameter have a default value definied. Those this makes sense?
But it could be a named parameter also.

```
async def create_report(
    report_name: str,
    force: bool = False,
    bpolys: Optional[FeatureCollection] = None,
    dataset: Optional[str] = None,
    feature_id: Optional[Union[int, str]] = None,
    fid_field: Optional[str] = None,
```